classes/rdkv-community-configs.bbclass

```diff
@@ -83,3 +83,20 @@ ctrlm_community_remote_fix() {
         bbnote "Detected default RCU Control manager configurations, skipping Community RCU Control manager configuration."
     fi
 }
+
+# Enable Miracast ports based on distro
+ROOTFS_POSTPROCESS_COMMAND:append = "${@bb.utils.contains('DISTRO_FEATURES', 'ENABLE_MIRACAST', ' update_ports_in_iptables; ', '', d)}"
+update_ports_in_iptables() {
+    if [ -f "${IMAGE_ROOTFS}/lib/rdk/iptables_init" ]; then
+sed -i "/${IPV4_BIN} -N SSHDROPLOG/i \\
+    # MiracastService plugin need to communicate with client through below ports\\

```

Comment:
Is the comments intentionally getting added into `{IMAGE_ROOTFS}/lib/rdk/iptables_init` ?
Please move the comments outside if not required. Also keep alignment proper.

Reply:
As per discussion; the practice is to add the comments in conf file.
Hence marking as resolved.